homeassistant/components/emoncms/sensor.py

```diff
@@ -198,22 +206,22 @@ def __init__(
             self._state = None
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return self._name
 
     @property
-    def native_unit_of_measurement(self):
+    def native_unit_of_measurement(self) -> str | None:
```

Comment:
Can you use shorthand attributes instead: `self._attr_unit_of_measurement`?

Reply:
used _attr_native_unit_of_measurement, hope I am not mistaken